rpcd/etc/openstack_deploy/user_osa_variables_defaults.yml

```diff
@@ -73,6 +73,7 @@ rabbitmq_ulimit: 65535
 # since memory utilization is primarily driven by the objects placed in memcached.
 memcached_connections: 16384
 
+#TODO (evrardjp): Remove this when ocata
 # Keystone overrides
 keystone_token_driver: "sql"
```

Comment:
@mancdaz yes indeed. So no need to ship the variable by default, except if the official stance currently (until O is released) is "When not using fernet, you must be using UUID + SQL tokens, else you'd be alone".